### PR TITLE
Improve the setup guide

### PR DESCRIPTION
**Pull request by jessicafalk**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Describe your changes here.
#### Tests
How did you test these changes prior to submitting this pull request?
What automated tests are included in this PR?
#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?
#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/setup-and-installing.md [hunk 1]
```diff
@@ -83,20 +83,20 @@ If you are using Rider, install the [**Unity Support** plugin](https://github.co
 
 <%(/Expandable)%>
 
-### Validate your installation
```
Comment (jessicafalk) [inline docs/setup-and-installing.md]: removing the validate section as `spatial diagnose` is useless for us. added some extra links for how to get help instead

File: docs/setup-and-installing.md [hunk 2]
```diff
@@ -83,20 +83,20 @@ If you are using Rider, install the [**Unity Support** plugin](https://github.co
...
+<%(#Expandable title="Having problems?")%>
```
Comment (jamiebrynes7) [inline docs/setup-and-installing.md]: Would be nice if we could offer some pointers on how to debug their own problems

Comment (jessicafalk) [reply docs/setup-and-installing.md]: I can add a link to the troubleshooting page?

Comment (jamiebrynes7) [reply docs/setup-and-installing.md]: If its potentially useful for setup problems

Comment (jessicafalk) [reply docs/setup-and-installing.md]: not really (at least right now)

but don't know what else to add

**Pull request merged**